### DISCUSSION TITLE
What is the recommended way to send non-auth email notifications to users?

### DISCUSSION THREAD

**Discussion by sbutler-gh** [supabase/supabase#1194] [Questions] [answered]
Is there an API in Supabase that can be used to send a non-auth email notification to users?

For example, a registered user named `Amy` creates a blog post.  A registered user named `Bob` comments on Amy's post.

I want to send Amy an email notification that her blog has received a new comment.

Is there any functionality in Supabase that enables this?

If so, can someone point to documentation?

If not, can someone recommend a good tool for sending email notifications would play well with Supabase and Svelte?

**Comment by kiwicopple** [comment, marked as answer]
We don't offer any transactional email services, so you need to use a 3rd party tool for this right now. A few off the top of my head:

- SES
- Mailgun
- OneSignal
- Sendgrid
- Mailchimp

We are also building a [Workflow engine](https://supabase.io/blog/2021/04/02/supabase-workflows) which will make these integrations very simple. I'd love to know which tool you use - perhaps we can build an integration for the Workflow engine

**Reply by sbutler-gh** [reply]
Got it, thanks so much for the input!

After doing some research, it appears there are several promising open-source transactional email services:

* https://postal.atech.media/
* https://cuttlefish.io/
* https://www.mailersend.com/mailgun-alternative

Postal and Cuttlefish are self-hosted, but Mailersend seems more of an open-source replacement for Mailgun — you can self-host Mailersend, but they also have a managed service offering.  So for simplicity, the Mailersend managed service is probably what I'll go with.

They also seem to have a more competitive offering (lower pricing, more features, better free tier) than Mailgun and Sendgrid:

https://www.mailersend.com/mailgun-alternative

**Reply by sbutler-gh** [reply]
On the topic, if any of these open-source transactional email services could be integrated into Supabase, that could be amazing!

Workflow will be a great step in that direction.  Still, if there was a transactional email functionality in the Supabase UI, that would pretty much be a full backend in one beautiful package!
